libs/openchallenges/ui/src/lib/search-dropdown-filter/search-dropdown-filter.component.html

```diff
@@ -44,32 +42,17 @@
     </ng-template>
   </ng-template>
 
-  <!-- Empty results templates -->
+  <!-- custom message for empty results -->
```

Comment:
What is a "message"?

Reply:
The message for empty results is "No results found".